package.json

```diff
@@ -48,11 +48,11 @@
     "docker-toolbelt": "^3.3.7",
     "dockerode": "^2.5.8",
     "lodash": "^4.17.4",
-    "resin-bundle-resolve": "^4.0.0",
+    "resin-bundle-resolve": "^4.1.0-35-alternative-template-names-dba0ecea12671ab1793e3d6d3b40ca44418dbb26",
```

Comment:
Please review the `resin-bundle-resolve` PR so I can update this line... :-)
https://github.com/balena-io-modules/resin-bundle-resolve/pull/36